vignettes/download_github_comments.Rmd

```diff
@@ -218,9 +222,9 @@ head(all_issues_combined) %>%
 
 The Refresh parser also parses issue data but the folder 'refresh_issues', parsing data retrieved from the search endpoint. In this notebook, that is data from the REFRESH issues portion saved to issue_search directory. 
```

Comment:
What is that first sentence trying to convey? The two parts seemingly don't correlate? 
1) The Refresh parser also parses issue data but the folder 'refresh_issues'
2) Parsing data retrieved from the search endpoint
Neither of these make sense alone, nor together. Unless the second is describing 'refresh_issues', it still does not make sense EVEN if we ignore it.

Also, do not meta-refer to the notebook like in "REFRESH issues portion". Find some way to say that the data retrieved from the ran functions is saved to issue_search directory.